lib/payload.json

```diff
@@ -181,7 +181,7 @@
             "contactState": {
               "type": "string",
               "description": "State of a contact sensor",
-              "enum": ["OPEN", "CLOSE"]
+              "enum": ["OPEN", "CLOSED"]
```

Comment:
And now you need to update the normalizer(s) and map the values correctly.

Reply:
I already changed everything from `doorState` to `contactState` and there are no CLOSE states anywhere.